Community updates

Apollon77 opened the pull request:
Hey, because of inactivity and because we were not able to reach you at alll one developer had forked your adapter into the Community-Repoand did needed adjustments for the adapter to still work.

We will now also release a new version. Please sync with us when you come back and can maintain the adapter again.

Ingo

halloamt commented:
Hi,

perfect, I do not have the time to maintain it anymore :( Can I give you acces to the npm repo?

Thanks,
Justus

pdbjjens commented:
@halloamt
Thank you for your consent to transfer your adapter to iobroker-community-adapters. Currently it seems that no further action is required by you for the transfer to npm. 
As @Apollon77 mentioned, we will release a new version with some bugfixes, js-controller adaptations and dependency updates. You will find it [here](https://github.com/iobroker-community-adapters/ioBroker.frontier_silicon)
I will also announce this in the ioBroker Forum [https://forum.iobroker.net/topic/38177/neuer-adapter-undok-fsapi](https://forum.iobroker.net/topic/38177/neuer-adapter-undok-fsapi).

dependabot[bot] committed:
Bump axios from 1.6.7 to 1.6.8 (#195)

dependabot[bot] committed:
Bump typescript from 5.3.3 to 5.4.5 (#198)

pdbjjens committed:
Merge pull request #194 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/testing-4.1.1

Bump @iobroker/testing from 4.1.0 to 4.1.1

dependabot[bot] committed:
Bump mocha from 10.3.0 to 10.4.0 (#196)

pdbjjens committed:
Merge pull request #197 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-20.12.2

Bump @types/node from 20.11.24 to 20.12.2

dependabot[bot] committed:
Bump @iobroker/testing from 4.1.1 to 4.1.3

Bumps [@iobroker/testing](https://github.com/AlCalzone/testing) from 4.1.1 to 4.1.3.
- [Changelog](https://github.com/ioBroker/testing/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/testing/compare/v4.1.1...v4.1.3)

---
updated-dependencies:
- dependency-name: "@iobroker/testing"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

mcm1957 committed:
Merge pull request #201 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/testing-4.1.3

Bump @iobroker/testing from 4.1.1 to 4.1.3

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.0.6 to 3.1.0

Bumps [@iobroker/adapter-core](https://github.com/ioBroker/adapter-core) from 3.0.6 to 3.1.0.
- [Release notes](https://github.com/ioBroker/adapter-core/releases)
- [Commits](https://github.com/ioBroker/adapter-core/compare/v3.0.6...v3.1.0)

---
updated-dependencies:
- dependency-name: "@iobroker/adapter-core"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 20.12.2 to 20.12.7 (#200)

pdbjjens committed:
Merge pull request #199 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/adapter-core-3.1.0

Bump @iobroker/adapter-core from 3.0.6 to 3.1.0

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.1.0 to 3.1.4 (#204)

dependabot[bot] committed:
Bump chai-as-promised from 7.1.1 to 7.1.2 (#206)

dependabot[bot] committed:
Bump @types/node from 20.12.7 to 20.13.0 (#208)

dependabot[bot] committed:
Bump sinon from 17.0.1 to 18.0.0

Bumps [sinon](https://github.com/sinonjs/sinon) from 17.0.1 to 18.0.0.
- [Release notes](https://github.com/sinonjs/sinon/releases)
- [Changelog](https://github.com/sinonjs/sinon/blob/main/docs/changelog.md)
- [Commits](https://github.com/sinonjs/sinon/compare/v17.0.1...v18.0.0)

---
updated-dependencies:
- dependency-name: sinon
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump typescript from 5.4.5 to 5.5.2 (#213)

dependabot[bot] committed:
Bump @types/node from 20.13.0 to 20.14.9 (#215)

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.1.4 to 3.1.6 (#217)

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-iobroker from 3.7.0 to 3.7.2 (#212)

dependabot[bot] committed:
Bump chai from 4.4.1 to 4.5.0 (#218)

dependabot[bot] committed:
Bump @alcalzone/release-script from 3.7.0 to 3.8.0 (#222)

dependabot[bot] committed:
Bump typescript from 5.5.2 to 5.5.4 (#223)

pdbjjens committed:
Merge pull request #209 from iobroker-community-adapters/dependabot/npm_and_yarn/sinon-18.0.0

Bump sinon from 17.0.1 to 18.0.0

dependabot[bot] committed:
Bump mocha and @types/mocha

Bumps [mocha](https://github.com/mochajs/mocha) and [@types/mocha](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/mocha). These dependencies needed to be updated together.

Updates `mocha` from 10.4.0 to 10.7.3
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v10.4.0...v10.7.3)

Updates `@types/mocha` from 10.0.6 to 10.0.7
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/mocha)

---
updated-dependencies:
- dependency-name: mocha
  dependency-type: direct:development
  update-type: version-update:semver-minor
- dependency-name: "@types/mocha"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #226 from iobroker-community-adapters/dependabot/npm_and_yarn/multi-3831629c3c

Bump mocha and @types/mocha

dependabot[bot] committed:
Bump @types/node from 20.14.9 to 22.2.0

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 20.14.9 to 22.2.0.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #227 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-22.2.0

Bump @types/node from 20.14.9 to 22.2.0

dependabot[bot] committed:
Bump axios from 1.6.8 to 1.7.4

Bumps [axios](https://github.com/axios/axios) from 1.6.8 to 1.7.4.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.6.8...v1.7.4)

---
updated-dependencies:
- dependency-name: axios
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #228 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.7.4

Bump axios from 1.6.8 to 1.7.4

pdbjjens committed:
2024H2 maintenance

pdbjjens committed:
Change unreachable retry #191

pdbjjens committed:
Cyclic connection retry

pdbjjens committed:
Beautify README

pdbjjens committed:
Merge pull request #229 from pdbjjens/master

2024H2 maintenance

pdbjjens committed:
chore: release v0.3.0

- (pdbjjens) Change: node>=18, js-contoller>=5 and admin>=6 required
- (pdbjjens) Change: Removed .npmignore
- (pdbjjens) Change: Cyclic connection retry instead of disabling the adapter (#191)
- (pdbjjens) New: Updated dependencies
- (pdbjjens) Fix: Replace deprecated method "deleteChannel" by "delObject" (#224)

dependabot[bot] committed:
Bump @types/node from 22.2.0 to 22.5.1 (#230)

dependabot[bot] committed:
Bump axios from 1.7.4 to 1.7.7 (#232)

pdbjjens committed:
Add node 22 testing

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.1.6 to 3.2.1

Bumps [@iobroker/adapter-core](https://github.com/ioBroker/adapter-core) from 3.1.6 to 3.2.1.
- [Release notes](https://github.com/ioBroker/adapter-core/releases)
- [Commits](https://github.com/ioBroker/adapter-core/compare/v3.1.6...v3.2.1)

---
updated-dependencies:
- dependency-name: "@iobroker/adapter-core"
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 22.5.1 to 22.7.4 (#237)

dependabot[bot] committed:
Bump typescript from 5.5.4 to 5.6.2 (#239)

dependabot[bot] committed:
Bump @types/mocha from 10.0.7 to 10.0.8 (#240)

dependabot[bot] committed:
Bump sinon from 18.0.0 to 19.0.2

Bumps [sinon](https://github.com/sinonjs/sinon) from 18.0.0 to 19.0.2.
- [Release notes](https://github.com/sinonjs/sinon/releases)
- [Changelog](https://github.com/sinonjs/sinon/blob/main/docs/changelog.md)
- [Commits](https://github.com/sinonjs/sinon/compare/v18.0.0...v19.0.2)

---
updated-dependencies:
- dependency-name: sinon
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #234 from iobroker-community-adapters/dependabot/npm_and_yarn/sinon-19.0.2

Bump sinon from 18.0.0 to 19.0.2

dependabot[bot] committed:
Bump @iobroker/testing from 4.1.3 to 5.0.0

Bumps [@iobroker/testing](https://github.com/AlCalzone/testing) from 4.1.3 to 5.0.0.
- [Changelog](https://github.com/ioBroker/testing/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/testing/compare/v4.1.3...v5.0.0)

---
updated-dependencies:
- dependency-name: "@iobroker/testing"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #235 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/adapter-core-3.2.1

Bump @iobroker/adapter-core from 3.1.6 to 3.2.1

pdbjjens committed:
Merge pull request #238 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/testing-5.0.0

Bump @iobroker/testing from 4.1.3 to 5.0.0

pdbjjens committed:
Add "stop"  media control

pdbjjens committed:
cleanup roles

pdbjjens committed:
change role of preset name

pdbjjens committed:
Merge pull request #242 from pdbjjens/master

Add media control STOP

pdbjjens committed:
Added button state acknowledgement

pdbjjens committed:
Add ack for volumeUp volumeDown

pdbjjens committed:
set button default to false

pdbjjens committed:
Merge pull request #243 from pdbjjens/master

Fix button state acknowledgement

pdbjjens committed:
removed close info.connection

pdbjjens committed:
Update Readme

pdbjjens committed:
Responsive Design Optimizations

pdbjjens committed:
Add network settings tab

pdbjjens committed:
Merge pull request #245 from pdbjjens/master

Responsive Design Optimizations

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.2.1 to 3.2.2 (#247)

dependabot[bot] committed:
Bump @types/node from 22.7.4 to 22.8.6 (#248)

dependabot[bot] committed:
Bump typescript from 5.6.2 to 5.6.3 (#249)

dependabot[bot] committed:
Bump mocha and @types/mocha

Bumps [mocha](https://github.com/mochajs/mocha) and [@types/mocha](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/mocha). These dependencies needed to be updated together.

Updates `mocha` from 10.7.3 to 10.8.2
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v10.7.3...v10.8.2)

Updates `@types/mocha` from 10.0.8 to 10.0.9
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/mocha)

---
updated-dependencies:
- dependency-name: mocha
  dependency-type: direct:development
  update-type: version-update:semver-minor
- dependency-name: "@types/mocha"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #250 from iobroker-community-adapters/dependabot/npm_and_yarn/multi-bd7f08ee8b

Bump mocha and @types/mocha

dependabot[bot] committed:
bump @types/node from 22.8.6 to 22.10.1 (#254)

dependabot[bot] committed:
bump axios from 1.7.7 to 1.7.8 (#257)

dependabot[bot] committed:
bump typescript from 5.6.3 to 5.7.2 (#260)

dependabot[bot] committed:
Bump @types/node from 22.10.1 to 22.10.3 (#262)

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.2.2 to 3.2.3 (#263)

dependabot[bot] committed:
Bump mocha from 10.8.2 to 11.0.1

Bumps [mocha](https://github.com/mochajs/mocha) from 10.8.2 to 11.0.1.
- [Release notes](https://github.com/mochajs/mocha/releases)
- [Changelog](https://github.com/mochajs/mocha/blob/main/CHANGELOG.md)
- [Commits](https://github.com/mochajs/mocha/compare/v10.8.2...v11.0.1)

---
updated-dependencies:
- dependency-name: mocha
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump axios from 1.7.8 to 1.7.9 (#266)

pdbjjens committed:
Merge pull request #264 from iobroker-community-adapters/dependabot/npm_and_yarn/mocha-11.0.1

Bump mocha from 10.8.2 to 11.0.1

dependabot[bot] committed:
bump chai-as-promised and @types/chai-as-promised

---
updated-dependencies:
- dependency-name: chai-as-promised
  dependency-type: direct:development
  update-type: version-update:semver-major
- dependency-name: "@types/chai-as-promised"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #261 from iobroker-community-adapters/dependabot/npm_and_yarn/multi-a7b1dbc0cd

bump chai-as-promised and @types/chai-as-promised

dependabot[bot] committed:
Bump typescript from 5.7.2 to 5.7.3 (#269)

dependabot[bot] committed:
Bump mocha from 11.0.1 to 11.1.0 (#268)

dependabot[bot] committed:
Bump @iobroker/testing from 5.0.0 to 5.0.3 (#271)

dependabot[bot] committed:
Bump @types/node from 22.10.3 to 22.13.0 (#270)

pdbjjens committed:
Migration to ESLint 9

pdbjjens committed:
fix merge conflicts

pdbjjens committed:
chore: release v0.4.0

- (pdbjjens) Change: media state changed from number to string and readonly (#241)
- (pdbjjens) New: Added media control function "stop" (#241)
- (pdbjjens) New: Optimizations for responsive design (#244)
- (pdbjjens) Change: Migration to ESLint 9 (#253)
- (pdbjjens) Fix: Added button state acknowledgement
- (pdbjjens) Fix: Prevent warning on adapter stop

pdbjjens committed:
Fix ESLint/Prettier problems

pdbjjens committed:
Merge pull request #272 from pdbjjens/master

Fix ESLint/Prettier problems

pdbjjens committed:
Update release notes v0.4.0

pdbjjens committed:
update required node, admin, js-controller

dependabot[bot] committed:
bump axios from 1.7.9 to 1.8.1

---
updated-dependencies:
- dependency-name: axios
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
bump @types/sinon from 17.0.3 to 17.0.4 (#274)

dependabot[bot] committed:
bump @iobroker/adapter-dev from 1.3.0 to 1.4.0 (#279)

dependabot[bot] committed:
Bump @types/node from 22.13.0 to 22.13.8 (#280)

pdbjjens committed:
Merge pull request #275 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.8.1

bump axios from 1.7.9 to 1.8.1

Commit:
Added dailight saving time

dependabot[bot] committed:
Bump @types/chai-as-promised from 8.0.1 to 8.0.2 (#283)

dependabot[bot] committed:
Bump axios from 1.8.1 to 1.8.4 (#284)

dependabot[bot] committed:
Bump @types/node from 22.13.8 to 22.13.14 (#288)

dependabot[bot] committed:
Bump sinon from 19.0.2 to 20.0.0

Bumps [sinon](https://github.com/sinonjs/sinon) from 19.0.2 to 20.0.0.
- [Release notes](https://github.com/sinonjs/sinon/releases)
- [Changelog](https://github.com/sinonjs/sinon/blob/main/docs/changelog.md)
- [Commits](https://github.com/sinonjs/sinon/compare/v19.0.2...v20.0.0)

---
updated-dependencies:
- dependency-name: sinon
  dependency-version: 20.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Fix UpdatePreset

pdbjjens committed:
Move documentation to docs  folder

pdbjjens committed:
Update states.md

pdbjjens committed:
Merge pull request #290 from pdbjjens/master

Fix UpdatePresets and rearrange documentation

pdbjjens committed:
Merge pull request #285 from iobroker-community-adapters/dependabot/npm_and_yarn/sinon-20.0.0

Bump sinon from 19.0.2 to 20.0.0

dependabot[bot] committed:
Bump @iobroker/testing from 5.0.3 to 5.0.4 (#287)

dependabot[bot] committed:
Bump @iobroker/eslint-config from 1.0.0 to 2.0.1

Bumps [@iobroker/eslint-config](https://github.com/ioBroker/ioBroker.eslint-config) from 1.0.0 to 2.0.1.
- [Release notes](https://github.com/ioBroker/ioBroker.eslint-config/releases)
- [Commits](https://github.com/ioBroker/ioBroker.eslint-config/compare/v1.0.0...v2.0.1)

---
updated-dependencies:
- dependency-name: "@iobroker/eslint-config"
  dependency-version: 2.0.1
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge branch 'master' into master

pdbjjens committed:
Merge pull request #286 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/eslint-config-2.0.1

Bump @iobroker/eslint-config from 1.0.0 to 2.0.1

pdbjjens committed:
Merge pull request #282 from oelison/master

Added daylight saving time

dependabot[bot] committed:
bump typescript from 5.7.3 to 5.8.2 (#277)

dependabot[bot] committed:
Bump axios from 1.8.4 to 1.9.0

Bumps [axios](https://github.com/axios/axios) from 1.8.4 to 1.9.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.8.4...v1.9.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.9.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 22.13.16 to 22.15.3 (#292)

pdbjjens committed:
Merge pull request #291 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.9.0

Bump axios from 1.8.4 to 1.9.0

pdbjjens committed:
Add node 24.x testing

dependabot[bot] committed:
Bump mocha from 11.1.0 to 11.5.0 (#294)

dependabot[bot] committed:
Bump @iobroker/eslint-config from 2.0.1 to 2.0.2 (#295)

dependabot[bot] committed:
Bump @types/node from 22.15.3 to 22.15.29 (#296)

dependabot[bot] committed:
Bump axios from 1.9.0 to 1.10.0

Bumps [axios](https://github.com/axios/axios) from 1.9.0 to 1.10.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.9.0...v1.10.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.10.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 22.15.29 to 24.0.8

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 22.15.29 to 24.0.8.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 24.0.8
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump mocha from 11.5.0 to 11.7.1 (#299)

pdbjjens committed:
Merge pull request #300 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-24.0.8

Bump @types/node from 22.15.29 to 24.0.8

dependabot[bot] committed:
Bump sinon from 20.0.0 to 21.0.0

Bumps [sinon](https://github.com/sinonjs/sinon) from 20.0.0 to 21.0.0.
- [Release notes](https://github.com/sinonjs/sinon/releases)
- [Changelog](https://github.com/sinonjs/sinon/blob/main/docs/changelog.md)
- [Commits](https://github.com/sinonjs/sinon/commits)

---
updated-dependencies:
- dependency-name: sinon
  dependency-version: 21.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #298 from iobroker-community-adapters/dependabot/npm_and_yarn/sinon-21.0.0

Bump sinon from 20.0.0 to 21.0.0

pdbjjens committed:
Merge pull request #297 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.10.0

Bump axios from 1.9.0 to 1.10.0

dependabot[bot] committed:
Bump axios from 1.10.0 to 1.11.0

Bumps [axios](https://github.com/axios/axios) from 1.10.0 to 1.11.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.10.0...v1.11.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.11.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 24.0.8 to 24.1.0 (#301)

pdbjjens committed:
Merge pull request #303 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.11.0

Bump axios from 1.10.0 to 1.11.0

pdbjjens committed:
cleanup devDependencies

pdbjjens committed:
fix dependencies

dependabot[bot] committed:
Bump actions/checkout from 4 to 5

Bumps [actions/checkout](https://github.com/actions/checkout) from 4 to 5.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #306 from iobroker-community-adapters/dependabot/github_actions/actions/checkout-5

Bump actions/checkout from 4 to 5

pdbjjens committed:
Merge pull request #305 from pdbjjens/master

cleanup devDependencies

pdbjjens committed:
Update dependencies

pdbjjens committed:
Merge pull request #307 from pdbjjens/master

Update dependencies

pdbjjens committed:
chore: release v0.5.0

- (pdbjjens) Change: node>=20, js-controller>=7.0.7 and admin>=7.6.17 required
- (oelison)  New: Read and write from daylight saving time
- (pdbjjens) Fix: UpdatePreset now handles empty presets correctly (#289)
- (pdbjjens) Change: Adapter and FSAPI documentation was moved to the docs folder (#281)
- (pdbjjens) Change: Cleanup devDependencies

dependabot[bot] committed:
Bump @iobroker/testing from 5.1.0 to 5.1.1 (#308)

Copilot committed:
Initial plan

Copilot committed:
feat: setup GitHub Copilot instructions and automation workflows

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

mcm1957 committed:
Merge pull request #312 from iobroker-community-adapters/copilot/fix-24018d38-3caf-4256-877f-af2d83705078

feat: Setup GitHub Copilot instructions and automation workflows

dependabot[bot] committed:
Bump actions/github-script from 7 to 8

Bumps [actions/github-script](https://github.com/actions/github-script) from 7 to 8.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v7...v8)

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-version: '8'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump actions/checkout from 4 to 5

Bumps [actions/checkout](https://github.com/actions/checkout) from 4 to 5.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v4...v5)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '5'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
bump @types/node from 24.3.0 to 24.6.1 (#315)

dependabot[bot] committed:
bump @iobroker/adapter-dev from 1.4.0 to 1.5.0 (#316)

dependabot[bot] committed:
bump axios from 1.11.0 to 1.12.2

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.12.2
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
bump typescript from 5.9.2 to 5.9.3 (#314)

dependabot[bot] committed:
bump @iobroker/eslint-config from 2.1.0 to 2.2.0 (#317)

pdbjjens committed:
Merge pull request #313 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.12.2

bump axios from 1.11.0 to 1.12.2

pdbjjens committed:
Merge pull request #319 from iobroker-community-adapters/dependabot/github_actions/actions/checkout-5

Bump actions/checkout from 4 to 5

pdbjjens committed:
Merge pull request #318 from iobroker-community-adapters/dependabot/github_actions/actions/github-script-8

Bump actions/github-script from 7 to 8

github-actions[bot] committed:
Update from template: X0000-MigrateToTrustedPublishing

github-actions[bot] committed:
Update from template: S9006-blockCommitinfo

pdbjjens committed:
Merge pull request #327 from ioBroker-Bot/update-from-template-S9006-blockCommitinfo-1760885523

[iobroker-bot] Block .commitinfo by adding to .gitignore

pdbjjens committed:
Merge pull request #325 from ioBroker-Bot/update-from-template-X0000-MigrateToTrustedPublishing-1760784986

[iobroker-bot] Migrate to NPM Trusted Publishing

pdbjjens committed:
Add npm Trusted Publisher

pdbjjens committed:
Fix typo

pdbjjens committed:
Update npm badge

pdbjjens committed:
Update npm badge

pdbjjens committed:
chore: release v0.5.1-alpha.0

- (pdbjjens) Fix: Add GitHub as npm Trusted Publisher
- (pdbjjens) Change: Update dependencies

pdbjjens committed:
Add JSON schemas Ref. issue #320  [S4038]

Copilot committed:
Initial plan

Copilot committed:
Update Copilot instructions from v0.4.0 to v0.4.2

Co-authored-by: UncleSamSwiss <7595126+UncleSamSwiss@users.noreply.github.com>

pdbjjens committed:
Merge pull request #328 from iobroker-community-adapters/copilot/update-iobroker-copilot-instructions

🤖 Update ioBroker Copilot Instructions to v0.4.2

dependabot[bot] committed:
Bump @alcalzone/release-script from 3.8.0 to 5.0.0

Bumps [@alcalzone/release-script](https://github.com/AlCalzone/release-script) from 3.8.0 to 5.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.8.0...v5.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script"
  dependency-version: 5.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 24.6.1 to 24.9.2 (#334)

pdbjjens committed:
Merge pull request #332 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-5.0.0

Bump @alcalzone/release-script from 3.8.0 to 5.0.0

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-iobroker from 3.7.2 to 4.0.0

Bumps [@alcalzone/release-script-plugin-iobroker](https://github.com/AlCalzone/release-script) from 3.7.2 to 4.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.7.2...v4.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-iobroker"
  dependency-version: 4.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #330 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-iobroker-4.0.0

Bump @alcalzone/release-script-plugin-iobroker from 3.7.2 to 4.0.0

dependabot[bot] committed:
Bump axios from 1.12.2 to 1.13.1

Bumps [axios](https://github.com/axios/axios) from 1.12.2 to 1.13.1.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.12.2...v1.13.1)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.13.1
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-manual-review from 3.7.0 to 4.0.0

Bumps [@alcalzone/release-script-plugin-manual-review](https://github.com/AlCalzone/release-script) from 3.7.0 to 4.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.7.0...v4.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-manual-review"
  dependency-version: 4.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #331 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-manual-review-4.0.0

Bump @alcalzone/release-script-plugin-manual-review from 3.7.0 to 4.0.0

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-license from 3.7.0 to 4.0.0

Bumps [@alcalzone/release-script-plugin-license](https://github.com/AlCalzone/release-script) from 3.7.0 to 4.0.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v3.7.0...v4.0.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-license"
  dependency-version: 4.0.0
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #329 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-license-4.0.0

Bump @alcalzone/release-script-plugin-license from 3.7.0 to 4.0.0

pdbjjens committed:
Merge pull request #333 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.13.1

Bump axios from 1.12.2 to 1.13.1

github-actions[bot] committed:
Update from template: X0000-randomDependabot

pdbjjens committed:
Merge pull request #335 from ioBroker-Bot/update-from-template-X0000-randomDependabot-1762694980

[iobroker-bot] Optimize Dependabot Schedule Configuration

dependabot[bot] committed:
Bump @iobroker/testing from 5.1.1 to 5.2.2 (#336)

dependabot[bot] committed:
Bump axios from 1.13.1 to 1.13.2 (#338)

dependabot[bot] committed:
Bump @types/node from 24.9.2 to 24.10.0 (#337)

dependabot[bot] committed:
Bump @types/node from 24.10.0 to 24.10.1 (#339)

dependabot[bot] committed:
Bump actions/checkout from 5 to 6

Bumps [actions/checkout](https://github.com/actions/checkout) from 5 to 6.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v5...v6)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 24.10.1 to 25.0.1

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 24.10.1 to 25.0.1.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 25.0.1
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #340 from iobroker-community-adapters/dependabot/github_actions/actions/checkout-6

Bump actions/checkout from 5 to 6

pdbjjens committed:
Merge pull request #341 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-25.0.1

Bump @types/node from 24.10.1 to 25.0.1

github-actions[bot] committed:
Update from template: E40xx-fixSchemaLinkForVsCode

pdbjjens committed:
Merge pull request #343 from ioBroker-Bot/update-from-template-E40xx-fixSchemaLinkForVsCode-1766271292

[iobroker-bot] Fix Schema Links for VSCode

github-actions[bot] committed:
Update from template: X0000-prepareCommunityAdapters

mcm1957 committed:
Merge pull request #344 from ioBroker-Bot/update-from-template-X0000-prepareCommunityAdapters-1767096865

[iobroker-bot] General update for iobroker-community-adapters repository

github-actions[bot] committed:
Update from template: X0000-setupAutomergeDependabot

pdbjjens committed:
Merge pull request #345 from ioBroker-Bot/update-from-template-X0000-setupAutomergeDependabot-1768514030

[iobroker-bot] Setup Auto-Merge for Dependabot PRs

pdbjjens committed:
implement navigation

pdbjjens committed:
Merge branch 'master' of https://github.com/pdbjjens/ioBroker.frontier_silicon

pdbjjens committed:
navigation of infinte lists

pdbjjens committed:
Sleep after mode switch to get new navList

pdbjjens committed:
Add currentNavListJson

pdbjjens committed:
Fixed some TypeErrors and extended currentNavIndex

github-actions[bot] committed:
Update from template: X0000-prepareCommunityAdapters

pdbjjens committed:
Add home and direct key navigation

mcm1957 committed:
Merge pull request #347 from ioBroker-Bot/update-from-template-X0000-prepareCommunityAdapters-1771850123

[iobroker-bot] General update for iobroker-community-adapters repository

Copilot committed:
Initial plan

Copilot committed:
chore: update copilot instructions from v0.4.2 to v0.5.7

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

mcm1957 committed:
Merge pull request #351 from iobroker-community-adapters/copilot/update-iobroker-copilot-instructions-another-one

Update ioBroker Copilot Instructions to v0.5.7

pdbjjens committed:
update release-script (#348)

github-actions[bot] committed:
Update from template: X0000-updateNodeJsAtTestAndRelease

mcm1957 committed:
Merge pull request #352 from ioBroker-Bot/update-from-template-X0000-updateNodeJsAtTestAndRelease-1772717661

[iobroker-bot] Update Node.js versions in test-and-release workflow

Copilot committed:
Initial plan

dependabot[bot] committed:
Bump axios from 1.13.2 to 1.13.6

Bumps [axios](https://github.com/axios/axios) from 1.13.2 to 1.13.6.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.13.2...v1.13.6)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.13.6
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

mcm1957 committed:
Update README with dependency updates and fixes

Updated README to reflect dependency changes and fixes.

github-actions[bot] committed:
Merge pull request #355 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.13.6

Bump axios from 1.13.2 to 1.13.6

Copilot committed:
fix: update check-copilot-template.yml with stable action versions and improved issue detection

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

Copilot committed:
security: upgrade axios to ^1.13.5 to patch DoS vulnerability (mergeConfig __proto__)

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

Copilot committed:
revert: restore package.json and package-lock.json to pre-PR state

Co-authored-by: mcm1957 <28575778+mcm1957@users.noreply.github.com>

mcm1957 committed:
Merge pull request #354 from iobroker-community-adapters/copilot/update-iobroker-copilot-instructions-yet-again

Fix recurring copilot-template check workflow

mcm1957 committed:
Update README.md

mcm1957 committed:
chore: release v0.5.1

- (copilot) Adapter requires admin >= 7.7.22 now
- (pdbjjens) **Fixed**: update release-script (#348)
- (mcm1957) Dependencies have been updated
- (pdbjjens) Fix: Add GitHub as npm Trusted Publisher
- (pdbjjens) Change: Update dependencies

pdbjjens committed:
Improved playlist tracking

pdbjjens committed:
some cleanup

pdbjjens committed:
optional navLogging

pdbjjens committed:
add set preset

pdbjjens committed:
Merge pull request #358 from pdbjjens/master

add navigation functions #342

mcm1957 committed:
add concurrency handling

mcm1957 committed:
Merge branch 'master' of https://github.com/iobroker-community-adapters/ioBroker.frontier_silicon

mcm1957 committed:
Delete dependabot-auto-merge.yml.OLD

github-actions[bot] committed:
Update from template: X0000-randomDependabot

pdbjjens committed:
update docs

pdbjjens committed:
Merge pull request #360 from pdbjjens/master

update docs

mcm1957 committed:
Merge pull request #359 from ioBroker-Bot/update-from-template-X0000-randomDependabot-1773581525

[iobroker-bot] Optimize Dependabot Schedule Configuration

pdbjjens committed:
modify loadChunkForNavName

pdbjjens committed:
fix loadChunkForNavName for tracks

pdbjjens committed:
Merge pull request #362 from pdbjjens/master

changed loadChunkForNavName

github-actions[bot] committed:
Update from template: S5043-useNodePrefix

pdbjjens committed:
Merge pull request #363 from ioBroker-Bot/update-from-template-S5043-useNodePrefix-1774710579

[iobroker-bot] Add node: prefix to Node.js built-in module requires and imports

github-actions[bot] committed:
Update from template: X0000-updateDependabotSettings

pdbjjens committed:
Merge pull request #364 from ioBroker-Bot/update-from-template-X0000-updateDependabotSettings-1774998451

[iobroker-bot] Update Dependabot Configuration – Add npm Cooldown

github-actions[bot] committed:
Update from template: X0000-updateNodeJsAtTestAndRelease

dependabot[bot] committed:
Bump actions/github-script from 8 to 9

Bumps [actions/github-script](https://github.com/actions/github-script) from 8 to 9.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v8...v9)

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-version: '9'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump axios from 1.13.6 to 1.14.0

Bumps [axios](https://github.com/axios/axios) from 1.13.6 to 1.14.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.13.6...v1.14.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.14.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #366 from ioBroker-Bot/update-from-template-X0000-updateNodeJsAtTestAndRelease-1775502826

[iobroker-bot] Update Node.js versions in test-and-release workflow

pdbjjens committed:
Merge pull request #368 from iobroker-community-adapters/dependabot/github_actions/actions/github-script-9

Bump actions/github-script from 8 to 9

pdbjjens committed:
Delete admin/style.css

admin/style.css" is outdated since jsonConfig is used

pdbjjens committed:
Merge pull request #369 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.14.0

Bump axios from 1.13.6 to 1.14.0

github-actions[bot] committed:
Update from template: X0000-dropNode20

pdbjjens committed:
Merge pull request #371 from ioBroker-Bot/update-from-template-X0000-dropNode20-1777476317

[iobroker-bot] Drop Node.js 20 support (End of Life)

github-actions[bot] committed:
Update from template: W8917-addDependabot-ignoreTypesNode

mcm1957 committed:
Merge pull request #372 from ioBroker-Bot/update-from-template-W8917-addDependabot-ignoreTypesNode-1777636653

[iobroker-bot] Add Dependabot Ignore Rule for @types/node Major Version Updates

dependabot[bot] committed:
Bump axios from 1.14.0 to 1.15.2

Bumps [axios](https://github.com/axios/axios) from 1.14.0 to 1.15.2.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.14.0...v1.15.2)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.15.2
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

pdbjjens committed:
Merge pull request #374 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.15.2

Bump axios from 1.14.0 to 1.15.2

github-actions[bot] committed:
Update from template: S6020-addChangelogOld

pdbjjens committed:
Merge pull request #375 from ioBroker-Bot/update-from-template-S6020-addChangelogOld-1777807733

[iobroker-bot] Add CHANGELOG_OLD.md to store older changelog entries

dependabot[bot] committed:
Bump axios from 1.15.2 to 1.16.0

Bumps [axios](https://github.com/axios/axios) from 1.15.2 to 1.16.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.15.2...v1.16.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.16.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Update from template: X0000-setupAutomergeIobrokerBot

mcm1957 committed:
Merge pull request #377 from ioBroker-Bot/update-from-template-X0000-setupAutomergeIobrokerBot-1778756640

[iobroker-bot] Setup Auto-Merge for ioBroker-Bot PRs

github-actions[bot] committed:
Update from template: X0000-setupAutomergeIobrokerBot

pdbjjens committed:
Merge pull request #378 from ioBroker-Bot/update-from-template-X0000-setupAutomergeIobrokerBot-1778797744

[iobroker-bot] Setup Auto-Merge for ioBroker-Bot PRs

pdbjjens committed:
Merge pull request #376 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.16.0

Bump axios from 1.15.2 to 1.16.0

github-actions[bot] committed:
Update from template: X0000-updateTypesNode

ioBroker-Bot committed:
Merge pull request #380 from ioBroker-Bot/update-from-template-X0000-updateTypesNode-1779344610

[iobroker-bot] Update @types/node to match minimum supported Node.js version

github-actions[bot] committed:
Update from template: W0086-updateTsconfigNode

ioBroker-Bot committed:
Merge pull request #382 from ioBroker-Bot/update-from-template-W0086-updateTsconfigNode-1780105880

[iobroker-bot] Align @tsconfig/nodeXX with engines.node in package.json

github-actions[bot] committed:
Update from template: X0000-removeCopilot

ioBroker-Bot committed:
Merge pull request #383 from ioBroker-Bot/update-from-template-X0000-removeCopilot-1780404496

[iobroker-bot] Remove Copilot setup and workflows

dependabot[bot] committed:
Bump ioBroker/testing-action-check from 1 to 2

Bumps [ioBroker/testing-action-check](https://github.com/iobroker/testing-action-check) from 1 to 2.
- [Release notes](https://github.com/iobroker/testing-action-check/releases)
- [Commits](https://github.com/iobroker/testing-action-check/compare/v1...v2)

---
updated-dependencies:
- dependency-name: ioBroker/testing-action-check
  dependency-version: '2'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-manual-review from 5.1.1 to 5.2.0

Bumps [@alcalzone/release-script-plugin-manual-review](https://github.com/AlCalzone/release-script) from 5.1.1 to 5.2.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v5.1.1...v5.2.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-manual-review"
  dependency-version: 5.2.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @iobroker/eslint-config from 2.2.0 to 2.3.4

Bumps [@iobroker/eslint-config](https://github.com/ioBroker/eslint-config) from 2.2.0 to 2.3.4.
- [Release notes](https://github.com/ioBroker/eslint-config/releases)
- [Commits](https://github.com/ioBroker/eslint-config/compare/v2.2.0...v2.3.4)

---
updated-dependencies:
- dependency-name: "@iobroker/eslint-config"
  dependency-version: 2.3.4
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump axios from 1.16.0 to 1.17.0

Bumps [axios](https://github.com/axios/axios) from 1.16.0 to 1.17.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.16.0...v1.17.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.17.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #388 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/eslint-config-2.3.4

Bump @iobroker/eslint-config from 2.2.0 to 2.3.4

github-actions[bot] committed:
Merge pull request #386 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-manual-review-5.2.0

Bump @alcalzone/release-script-plugin-manual-review from 5.1.1 to 5.2.0

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-license from 5.1.1 to 5.2.0

Bumps [@alcalzone/release-script-plugin-license](https://github.com/AlCalzone/release-script) from 5.1.1 to 5.2.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v5.1.1...v5.2.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-license"
  dependency-version: 5.2.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #387 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-license-5.2.0

Bump @alcalzone/release-script-plugin-license from 5.1.1 to 5.2.0

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-iobroker from 5.1.2 to 5.2.0

Bumps [@alcalzone/release-script-plugin-iobroker](https://github.com/AlCalzone/release-script) from 5.1.2 to 5.2.0.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v5.1.2...v5.2.0)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-iobroker"
  dependency-version: 5.2.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #385 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-iobroker-5.2.0

Bump @alcalzone/release-script-plugin-iobroker from 5.1.2 to 5.2.0

dependabot[bot] committed:
Bump @alcalzone/release-script from 5.1.1 to 5.2.1

Bumps [@alcalzone/release-script](https://github.com/AlCalzone/release-script) from 5.1.1 to 5.2.1.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v5.1.1...v5.2.1)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script"
  dependency-version: 5.2.1
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #389 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-5.2.1

Bump @alcalzone/release-script from 5.1.1 to 5.2.1

pdbjjens committed:
Merge pull request #384 from iobroker-community-adapters/dependabot/github_actions/ioBroker/testing-action-check-2

Bump ioBroker/testing-action-check from 1 to 2

pdbjjens committed:
Merge pull request #390 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.17.0

Bump axios from 1.16.0 to 1.17.0

dependabot[bot] committed:
Bump axios from 1.17.0 to 1.18.1

Bumps [axios](https://github.com/axios/axios) from 1.17.0 to 1.18.1.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.17.0...v1.18.1)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.18.1
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @types/node from 22.19.19 to 22.20.0

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 22.19.19 to 22.20.0.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 22.20.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #393 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-22.20.0

Bump @types/node from 22.19.19 to 22.20.0

pdbjjens committed:
Merge pull request #392 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.18.1

Bump axios from 1.17.0 to 1.18.1

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.3.2 to 3.4.1

Bumps [@iobroker/adapter-core](https://github.com/ioBroker/adapter-core) from 3.3.2 to 3.4.1.
- [Release notes](https://github.com/ioBroker/adapter-core/releases)
- [Commits](https://github.com/ioBroker/adapter-core/compare/v3.3.2...v3.4.1)

---
updated-dependencies:
- dependency-name: "@iobroker/adapter-core"
  dependency-version: 3.4.1
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

mcm1957 committed:
Merge pull request #394 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/adapter-core-3.4.1

Bump @iobroker/adapter-core from 3.3.2 to 3.4.1

dependabot[bot] committed:
Bump @types/node from 22.20.0 to 22.20.1

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 22.20.0 to 22.20.1.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-version: 22.20.1
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @iobroker/adapter-core from 3.4.1 to 3.4.3

Bumps [@iobroker/adapter-core](https://github.com/ioBroker/adapter-core) from 3.4.1 to 3.4.3.
- [Release notes](https://github.com/ioBroker/adapter-core/releases)
- [Commits](https://github.com/ioBroker/adapter-core/compare/v3.4.1...v3.4.3)

---
updated-dependencies:
- dependency-name: "@iobroker/adapter-core"
  dependency-version: 3.4.3
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @alcalzone/release-script-plugin-license from 5.2.0 to 5.2.2

Bumps [@alcalzone/release-script-plugin-license](https://github.com/AlCalzone/release-script) from 5.2.0 to 5.2.2.
- [Release notes](https://github.com/AlCalzone/release-script/releases)
- [Changelog](https://github.com/AlCalzone/release-script/blob/master/CHANGELOG.md)
- [Commits](https://github.com/AlCalzone/release-script/compare/v5.2.0...v5.2.2)

---
updated-dependencies:
- dependency-name: "@alcalzone/release-script-plugin-license"
  dependency-version: 5.2.2
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #395 from iobroker-community-adapters/dependabot/npm_and_yarn/types/node-22.20.1

Bump @types/node from 22.20.0 to 22.20.1

github-actions[bot] committed:
Merge pull request #396 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/adapter-core-3.4.3

Bump @iobroker/adapter-core from 3.4.1 to 3.4.3

dependabot[bot] committed:
Bump @iobroker/testing from 5.2.2 to 5.3.0

Bumps [@iobroker/testing](https://github.com/ioBroker/testing) from 5.2.2 to 5.3.0.
- [Release notes](https://github.com/ioBroker/testing/releases)
- [Changelog](https://github.com/ioBroker/testing/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ioBroker/testing/compare/v5.2.2...v5.3.0)

---
updated-dependencies:
- dependency-name: "@iobroker/testing"
  dependency-version: 5.3.0
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #400 from iobroker-community-adapters/dependabot/npm_and_yarn/alcalzone/release-script-plugin-license-5.2.2

Bump @alcalzone/release-script-plugin-license from 5.2.0 to 5.2.2

dependabot[bot] committed:
Bump axios from 1.18.1 to 1.19.0

Bumps [axios](https://github.com/axios/axios) from 1.18.1 to 1.19.0.
- [Release notes](https://github.com/axios/axios/releases)
- [Changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md)
- [Commits](https://github.com/axios/axios/compare/v1.18.1...v1.19.0)

---
updated-dependencies:
- dependency-name: axios
  dependency-version: 1.19.0
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge pull request #398 from iobroker-community-adapters/dependabot/npm_and_yarn/iobroker/testing-5.3.0

Bump @iobroker/testing from 5.2.2 to 5.3.0

pdbjjens committed:
Merge pull request #397 from iobroker-community-adapters/dependabot/npm_and_yarn/axios-1.19.0

Bump axios from 1.18.1 to 1.19.0
